man/systemd.link.xml

```diff
@@ -115,6 +115,21 @@
           property <varname>DEVTYPE</varname>.</para>
         </listitem>
       </varlistentry>
+      <varlistentry>
```

Comment:
Maybe move this to the very end of the [Match] options? It's a "catch all" option, and I think we should push people towards using the other possibilities. In particular, the rvalues in the other matches are well-defined and standardized, but udev properties not as much.

Reply:
`Type=` or `Driver=` correspond to udev properties and, thus, properties of the network interface.
However, `Host=` or `Virtualization=` do not correspond to udev property, and are system's condition.
So, I like the current position of `Property=`, that is, the end of the settings correspond to udev properties.